src/test/groovy/com/transferwise/common/baseutils/UuidUtilsTest.java

```diff
@@ -59,4 +59,9 @@ public void convertingFromBytesAndBackToUuidEndWithTheSameResult() {
     assertArrayEquals(expected, result);
   }
 
+  @Test
+  public void generatingSecuredUuidWorks() {
+    assertThat(UuidUtils.generateSecureUuid()).isNotNull();
```

Comment:
More of a comment than a request: wonder how we would test this further. Perhaps verify that SecureRandom was interacted with?